tests/it/fossa_cli.rs

```diff
@@ -0,0 +1,66 @@
+use std::path::PathBuf;
+
+use broker::{
+    config::RawBaseArgs,
+    fossa_cli::{self, DesiredVersion},
+};
+use tracing_test::traced_test;
+use uuid::Uuid;
+
+use crate::helper::temp_config;
+
+#[tokio::test]
+async fn downloads_latest_cli() {
```

Comment:
Very nice that we can add tests for this now that we're always downloading the latest CLI version. I should've realized that when I wrote that PR!